client/src/App.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { LineChart, Line, CartesianGrid, XAxis, YAxis, Tooltip } from 'recharts';
+import './App.css';
+
+const URL = 'http://localhost:5000/data'
+class App extends React.Component {
+  state = { data: [] }
+
+  componentDidMount() {
+    fetch(URL)
```

Comment:
Assuming that this application would grow:

1. Any idea how we could split "rendering" from actual data requests?  
2. If we were to "centralise" the data (e.g. use those "yield" values to render 20 indepedent graphs), how would you implement this?

Reply:
1. There'd be a separated file(s) with api requests - probaly using Axios with async/await instead of fetch.
2. I believe you mean using redux + thunk to keep the data in one place.

Reply:
I was aiming at the component-container pattern, but since you mentioned redux already, we can leave it at that.